packages/tasks_panel/src/utils/task-serializer.ts

```diff
@@ -1,5 +1,70 @@
 import { ConfiguredTask } from "@sap_oss/task_contrib_types";
+import { randomBytes } from "crypto";
+import escapeStringRegexp = require("escape-string-regexp");
+import { filter, map } from "lodash";
+import { getConfiguredTasksFromCache } from "../../src/services/tasks-provider";
+import { ConfigurationTarget, TaskDefinition, Uri, commands, languages, window, workspace } from "vscode";
 
 export function serializeTask(task: ConfiguredTask): string {
   return JSON.stringify(task);
 }
+
+export function generateUniqueCode(): string {
+  function generateGroup(): string {
+    return randomBytes(2).toString("hex").toUpperCase();
```

Comment:
do you really need a random 16bits hex code here?
Would an integer counter also work?
Note that an Integer counter would provide a much larger pool of options vs 16bits
```
> Number.MAX_SAFE_INTEGER
9007199254740991
```